pkgs/development/php-packages/rrd/default.nix

```diff
@@ -0,0 +1,17 @@
+{ buildPecl, lib, pkg-config, rrdtool }:
+
+buildPecl {
+  pname = "rrd";
+
+  version = "2.0.3";
+  sha256 = "sha256-pCFh5YzcioU7cs/ymJidy96CsPdkVt1ZzgKFTJK3MPc=";
+
+  nativeBuildInputs = [ pkg-config rrdtool ];
+
+  meta = {
+    description = "PHP bindings to rrd tool system";
+    license = lib.licenses.bsd0;
+    homepage = "https://github.com/php/pecl-processing-rrd";
+    maintainers = lib.teams.wdz.members;
```

Comment:
Is every member of the team OK to be listed as maintainers here?

Reply:
Yes, this is the team of NixOS people from our company and we maintain all our modules together that we need for our infrastructure.

Reply:
Great, I take your words for granted then!

Reply:
It's fine ✌🏻